src/libraries/System.Text.Json/src/System/Text/Json/Serialization/JsonPropertyInfoOfT.cs

```diff
@@ -11,12 +11,12 @@ namespace System.Text.Json
     /// <summary>
     /// Represents a strongly-typed property to prevent boxing and to create a direct delegate to the getter\setter.
     /// </summary>
-    internal sealed class JsonPropertyInfo<TTypeToConvert> : JsonPropertyInfo
+    internal sealed class JsonPropertyInfo<T> : JsonPropertyInfo
```

Comment:
Yes the double T ("TypeTypeToConvert") wasn't consistent with other areas that just use T (for the same usage).

Reply:
I added this comment

```csharp
/// <typeparamref name="T"/> is the <see cref="JsonConverter{T}.TypeToConvert"/> for either the property's converter,
/// or a type's converter, if the current instance is a <see cref="JsonClassInfo.PropertyInfoForClassInfo"/>.
```